### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by blaisexen** [mhoellerschlieper/DelphiCNN#2]
Question using this project for image recognition like Food-Recipe,

is there a builtin function name that we can train and predict image result?
for example train 1 image eq., burger and another 1 eq., spaghetti,
then predict if the compare image is identical to the trained images.

code, like 
to train image,
delphicnntrain(thisimage1, tothisgroup1);
delphicnntrain(thisimage2, tothisgroup1);
delphicnntrain(thisimage3, tothisgroup1);
delphicnntrain(thisimage4, tothisgroup1);

to predict image from compared image and accept confidence value greater than 95% only
result = delphicnnpredict(thisotherimageguesswhat, tothisgroup1);
0=unknown
1=found

how can we put the codes together to get the function like that we want? I guess thats a user friendly function.

**Comment by mhoellerschlieper** [comment]
Hi,

actually I do no have such a function to read an external file, but I will work on it. The next project uses the Tiny-Imges containig a lot of JPGs - thus I have to implement a function like you need. 
I think I need some weeks to develop the tinynet Interface. Have a look on my repositorities.

Best regards

Marcus

**Comment by blaisexen** [comment]
hi,

Actually I'm testing CNN for face recognition, and lot of links around the internet specially GitHub has the coolest project developed.

I have seen projects years ago and forgot the link, the project was using CNN too, and the other is PNN,, I don't know what's there differences But when I tried there binary was so cool.  For example, I only input 1 face image picture in the morning and in the afternoon I also tried to recognized from the web camera then it work like a charm, input face from picture then output result from webcam, it was so cool.  I'm so sure the other project is using PNN  too because I tested it also, they both have very good results.

I hope for this project too.

Yes, I have look your repositories, Very good Delphi projects, including your c++ project. 
thank you
